### ISSUE TITLE
a11y-view broken for local builds

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#667]
> http://localhost:8080/balloons-and-static-electricity/build/phet/balloons-and-static-electricity_en.html?postMessageOnLoad&postMessageOnError&accessibility 404 (Not Found)

**Comment by jessegreenberg** [comment]
Fixed with the above commit. There was a check that removed the 'phet' brand suffix for local builds,  I don't know why. Closing.

**Comment by zepumph** [comment]
potential solution: request one version in a `try` clause, and if you get back a 404, then try the other one. That way it will always work.

**Comment by jessegreenberg** [comment]
In phetsims/a11y-research#111 @zepumph also said
> at least we should add 'localhost' to that list.

**Comment by jessegreenberg** [comment]
@zepumph your reccomendation in https://github.com/phetsims/chipper/issues/667#issuecomment-397437363 actually seems totally sufficient for what we need in the a11y view.

**Comment by jessegreenberg** [comment]
https://stackoverflow.com/questions/16499117/how-to-detect-an-error-404-in-an-iframe/26641263

:(
> The 404 Page is handling an HTTP Status Code, which is only included in the server's response sent to the browser, but not in the actual window and document objects of the DOM that javascript may access.

**Comment by zepumph** [comment]
What if we just did an XHR on the desired URL, and then handled the error there and loaded the iframe accordingly? Like an ajax solution rather than dealing with the browser's iframe api.

**Comment by jessegreenberg** [comment]
Ah, very cool. I am not too sure about how that works but this looks like it could help:
https://stackoverflow.com/questions/10418031/how-do-i-capture-the-http-code-of-an-iframe/

**Comment by jessegreenberg** [comment]
@zepumph I gave it a shot in the above commit and it seems to be working OK. Can you please review? Seems to be working for RequireJS, local builds, and builds on bayes, but haven't been able to test a version on phettest yet (clicking the "build" button results in "failed").

The only down side is that if the `open` fails, we will get a 404 error in the console, though that isn't a huge deal.

**Comment by zepumph** [comment]
I agree, that is annoying, I hadn't thought about that. I guess you can't suppress that 404 from the server. I tried using a try/catch, and handling it in the on error, but no luck.

Instead I went with filling in a specific constant only on build, and it seems much simpler. @jessegreenberg what do you think?

**Comment by zepumph** [comment]
I tested local, local built, and https://bayes.colorado.edu/dev/html/balloons-and-static-electricity/1.4.0-dev.29/phet/balloons-and-static-electricity_a11y_view.html

**Comment by jessegreenberg** [comment]
Ah, that is really nice @zepumph, thanks. Much simpler and gets rid of the 404 error showing up in the console for us. Verified this works locally so closing.
